community/contributing/docs/local-clone.md

```diff
@@ -0,0 +1,263 @@
+   ```
+
+## Update your local repository
+
+Make sure your local repository is current before you start making changes. Fetch commits from your fork's `origin/master` and `spinnaker/spinnaker.github.io`'s `upstream/master`:
+
+   ```bash
+   git fetch origin
+   git fetch upstream
+   ```
+
+   This workflow is different from a source code commit workflow . You do not need to rebase your local copy of `master` with `upstream/master` before pushing updates to your fork. In this documentation workflow, you create a working branch that tracks changes to `upstream/master` rather than `origin/master`.
+
+## Create a working branch
+
+1. Create a new working branch based on `upstream/master`:
+
+   ```bash
+   git checkout -b <your-working-branch> upstream/master
+   ```
+
+1.  Make your changes.
+
+Use the `git status` command at any time to see what files you've changed.
+
+## Preview your changes locally
+
+It's a good idea to preview your changes locally before opening a pull request (PR). A preview lets you catch build errors or markdown formatting problems. The easiest way to deploy the documentation site is with a Docker container built using the included `Dockerfile`. Make sure you have [Docker](https://www.docker.com/get-started) installed on your computer. Run the following commands from the `spinnaker.github.io` directory:
+
+```bash
+docker build --tag spinnaker/spinnaker.github.io-test .
+docker run -it --rm --mount "type=bind,source=$(pwd),target=/code" \
+    -p 4000:4000 spinnaker/spinnaker.github.io-test --incremental
+```
+
+Alternately, if you have [Jekyll](https://jekyllrb.com/) installed, you can generate the site by executing:
+
+```bash
+bundle exec jekyll serve --watch --incremental
+```
+
+Navigate to `http://localhost:4000` to see your changes.
+
+## Commit your changes
+
+Commit your changes when you are ready to submit a pull request.
+
+1. Check which files you need to commit:
+
+   ```bash
+   git status
+   ```
+
+   Output is similar to:
+
+   ```bash
+   On branch local-clone
+   Changes not staged for commit:
+   (use "git add <file>..." to update what will be committed)
+   (use "git restore <file>..." to discard changes in working directory)
+
+   modified:   _data/navigation.yml
+
+   Untracked files:
+   (use "git add <file>..." to include in what will be committed)
+
+   community/contributing/docs/local-clone.md
+
+   no changes added to commit (use "git add" and/or "git commit -a")
+   ```
+
+1. Add new files listed under **Untracked files** to the commit:
+
+   ```bash
+   git add <your-file-name>
+   ```
+
+   Repeat this for each new file.
+
+1. Create a commit:
+
+   ```bash
+   git commit -a -m "<your-commit-subject>" -m "<your-commit-description>"
+   ```
+
+   - `-a`: Commits all changes.
+   - `-m`: Use the given `<msg>` as the commit message. If multiple `-m` options are given, their values are concatenated as separate paragraphs.
```

Comment:
In the command above, the `-m`s are followed by `<your-commit-subject>` and `<your-commit-description>`, so "...the given `<msg>`" here might be confusing.